### DISCUSSION TITLE
Pinning the rows

### DISCUSSION THREAD

**Discussion by talmacsi** [ThioJoe/ThioJoe-AHK-Scripts#6] [Ideas]
First of all: thanx for the app!!!
Never new, that I've been always missing this. I think It will help me a lot.
I think, with a possibility to pin rows, like in the clipboard hystory (win+v), would be the ultimate solution!
It could be stored in the settings.ini; and probably could be limited number of pinned paths.

**Comment by ThioJoe** [comment]
I'm planning on at least adding an option for pinned favorites that always show. I hadn't considered the clipboard history, but I'll have to see how hard it would be with AHK to have it retrieve the list of clipboard history items. Then maybe I could have it parse those items and show any paths like it does for the current clipboard item, but include any others from the history too.

**Reply by talmacsi** [reply]
Actually I've mentioned the clipboard history as an example of the simple pinning.
I don't think it would be a must. It would be good to implement that also; but a simple pinned favourites would be more than perfect!

**Reply by talmacsi** [reply]
Actually, I'm not quite familiar in github. Is this the better place for a suggestion like this or at the Issues?
